react-number-format/build.boot

```diff
@@ -0,0 +1,38 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]
+                  [cljsjs/react "15.3.1-0"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "2.0.0")
+(def +version+ (str +lib-version+ "-alpha2"))
+
+(task-options!
+ pom  {:project     'cljsjs/react-number-format
+       :version     +version+
+       :description "Yet another react component for input masking"
+       :url         "https://github.com/s-yadav/react-number-format"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(require '[boot.core :as c]
+         '[boot.tmpdir :as tmpd]
+         '[clojure.java.io :as io]
+         '[clojure.string :as string])
+
+(deftask package []
+  (comp
+   (download :url (str "https://github.com/s-yadav/react-number-format/archive/v2.0.0-alpha2.zip")
```

Comment:
@martinklepsch This is a text input component that allows for formatting currency as the user  enters in the text field.